chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
-appVersion: "1.26.1"
+appVersion: "1.27.2"
 description: Helm chart for fusionauth
 type: application
 name: fusionauth
-version: 0.0.0+dev
+version: 0.9.3
```

Comment:
Can you change this back? The chart here is not supposed to have a specific version number in it.

Reply:
done on 78bc8a5